protocols/perf/Cargo.toml

```diff
@@ -20,7 +20,7 @@ instant = "0.1.11"
 libp2p-core = { version = "0.39.0", path = "../../core" }
 libp2p-dns = { version = "0.39.0", path = "../../transports/dns", features = ["async-std"] }
 libp2p-identity = { version = "0.1.0", path = "../../identity" }
-libp2p-noise = { version = "0.42.0", path = "../../transports/noise" }
+libp2p-noise = { version = "0.42.2", path = "../../transports/noise" }
```

Comment:
I am assuming that you are not bumping the `libp2p-perf` version in this pull request as this change does not warrant a new release, i.e. it doesn't need to be included in the next round of patch releases. Is that correct?

Reply:
Same as above.